sr_robot_launch/scripts/construct_robot_description

```diff
@@ -35,8 +35,15 @@ class SrConstructRobotDescription():
         CONST_ARM_TYPE = rospy.get_param("~arm_type")
         self._side = ''
         self._robot_description_params = {}
-        self._CONST_RA_ARM = {'prefix': 'ra', 'ip_address': '10.8.1.1', 'arm_type': CONST_ARM_TYPE}
-        self._CONST_LA_ARM = {'prefix': 'la', 'ip_address': '10.8.2.1', 'arm_type': CONST_ARM_TYPE}
+        TIMEOUT_WAIT_FOR_PARAMS_IN_SECS = 60.0
+
+        self._right_arm_ip = '10.8.1.1'
+        self._left_arm_ip = '10.8.2.1'
+
+        self._wait_for_arm_ip_params(TIMEOUT_WAIT_FOR_PARAMS_IN_SECS)
+
+        self._right_arm_params = {'prefix': 'ra', 'ip_address': self._right_arm_ip, 'arm_type': CONST_ARM_TYPE}
```

Comment:
You are changing _CONST_RA_ARM -> _right_arm_params here but the rest of the code still uses old name _CONST_RA_ARM.

Reply:
addressed in https://github.com/shadow-robot/sr_interface/pull/586/commits/df44fa98bba485f979ea12e3ef69c766f53209fa